README.md

```diff
@@ -69,116 +70,115 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.58 - April 2022 Update
+### 0.59 - May 2022 Update
 
-In this release, we focused on upgrading to new dependencies and building for native ARM64.  Below are some of the highlights!
+In this release, we focused on wrapping up building for native ARM64 and releasing the first experimental build. Below are some of the highlights!
 
 **Highlights**
 
-- Most of the work for running natively on ARM64 has been included in the repo. Thanks [@snickler](https://github.com/snickler)!
-- Uses of the obsolete WebBrowser control were replaced with WebView 2.
-- All uses of .NET Core 3.1 were removed. PowerToys now runs on .NET 6.
-- Settings no longer runs on XAML Islands and is now running on WinUI 3, fixing many bugs related to XAML islands.
+- The work for running natively on ARM64 has been wrapped up and a build is released. Thanks [@snickler](https://github.com/snickler)!
+- Power Rename no longer runs on WinUI 2 and is now running on WinUI 3.
+- Keyboard Manager now allows up to 4 modifier keys for shortcuts and has received some quality fixes.
+
+### Known issues
+- There are reports of users who are [unable to open the Settings window](https://github.com/microsoft/PowerToys/issues/18015). This is being caused by incompatibilities with some applications in some cases (RTSS RivaTuner Statistics Server and MSI AfterBurner are known examples of this), or by some UAC settings which cause every application to run elevated. If you're affected by this, please check the  linked issue to verify if any of the presented solutions works for you.
+- Some users are [unable to install PowerToys](https://github.com/microsoft/PowerToys/issues/18021) due to the Windows App SDK runtime failing to install. If you're affected by this, please check the issue to verify if any of the presented solutions works for you.
+- PowerToys can't currently install on Windows 10 ARM64, due to the Windows App SDK installer dependency failing to install. We're aware of this and investigating a workaround until the Windows App SDK installer supports Windows 10 ARM64.
```

Comment:
So, "The work for running natively on ARM64 has been wrapped up and a build is released", but it can't be installed? That's really confusing...

Reply:
It can be installed on Windows 11, which seems to be where the majority of ARM64 users of Windows are already. I'll make it clearer.